d2themes/d2themescatalog/catalog.go

```diff
@@ -22,6 +22,7 @@ var Catalog = []d2themes.Theme{
 	EarthTones,
 	EvergladeGreen,
 	ButteredToast,
+	CatppuccinMochaMauve,
```

Comment:
i know it's the name of the theme but it's a really weird, unfriendly string of barely-english words. 

Any chance we can just name this like, "MochaDark" or something?

Reply:
Well `Mocha` already means dark in the context of catppuccin, is something like `CatppuccinDark` possible?

Reply:
still doesn't give me a sense of what to expect, unlike all the other themes. Just looks like a typo to me of Cappuccino lol. i'd still prefer MochaDark

Reply:
I understand, but I would prefer at least `DarkMauve` (dark - purple). Cuz `MochaDark` is like dark - dark.

Reply:
oh interesting i didn't know that was a color name. I'm cool with that!

Reply:
alright nice!